guides/catalog/02 Creating and Managing Products.ipynb

```diff
@@ -34,120 +36,93 @@
    "id": "c661c1ac-b9ce-4e18-a741-fe49f5149085",
    "metadata": {},
    "source": [
-    "## Creating and updating a product\n",
-    "Before uploading images to the catalog, you need to create a product and declare its bands. The only required attributes for a product are a unique id and a name:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "1829e262-a26e-494f-a9e2-65dc03ef7c26",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "# Creating a unique ID to avoid conflicts:\n",
-    "from uuid import uuid4\n",
+    "## Creating and updating a `Product`\n",
+    "The only required attributes for a `Product` are a unique id and a name. \n",
+    "\n",
+    "A `Product ID` should always have the following structure:\n",
+    "\n",
+    "    `my-org-id:my-product-id`\n",
```

Comment:
I think this is a bit misleading, right? We go on to recommend using a namespace as well. Also, doesn't the org get pre-pended without the user specifying it?